src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -4923,15 +4923,15 @@ T P
                 // (9,9): error CS8374: Cannot ref-assign 'tOut' to 'F' because 'tOut' has a narrower escape scope than 'F'.
                 //         F = ref tOut; // 2
                 Diagnostic(ErrorCode.ERR_RefAssignNarrower, "F = ref tOut").WithArguments("F", "tOut").WithLocation(9, 9),
-                // (10,17): error CS8331: Cannot assign to variable 'in T' because it is a readonly variable
+                // (10,17): error CS8329: Cannot use variable 'in T' as a ref or out value because it is a readonly variable
                 //         F = ref tIn; // 3
-                Diagnostic(ErrorCode.ERR_AssignReadonlyNotField, "tIn").WithArguments("variable", "in T").WithLocation(10, 17),
+                Diagnostic(ErrorCode.ERR_RefReadonlyNotField, "tIn").WithArguments("variable", "in T").WithLocation(10, 17),
                 // (16,13): error CS8374: Cannot ref-assign 'value' to 'F' because 'value' has a narrower escape scope than 'F'.
                 //             F = ref value; // 4
                 Diagnostic(ErrorCode.ERR_RefAssignNarrower, "F = ref value").WithArguments("F", "value").WithLocation(16, 13),
-                // (18,21): error CS8331: Cannot assign to method 'S<T>.GetRefReadonly()' because it is a readonly variable
+                // (18,21): error CS8329: Cannot use method 'S<T>.GetRefReadonly()' as a ref or out value because it is a readonly variable
```

Comment:
>Cannot use method 'S<T>.GetRefReadonly()' as a ref or out value because it is a readonly variable [](http://example.com/codeflow?start=42&length=97)

The message doesn't really make sense since it calls method a variable. It looks like the error that we previously reported also had this problem. I think we should consider adjusting the wording, For example, if we drop the argument that injects the symbol kind, the message might become less confusing.